Src/Sound/SoundStream.cpp

```diff
@@ -0,0 +1,204 @@
+// A Sound Stream is the thing that actually plays audio data like PGE::Sound through the speakers.
+#include "SoundStream.h"
+
+#include <Math/Math.h>
+#include <String/String.h>
+#include <Misc/FilePath.h>
+
+#include "PGESound.h"
+
+#include <../../Engine/Libraries/SDL2/include/SDL_audio.h>
+#include <../../Engine/Libraries/SDL2/include/SDL.h>
+
+#include <fstream>
+#include <map>
+#include <ogg/ogg.h>
+#include <vorbis/codec.h>
+//#include <vorbis/vorbisenc.h>
+#include <vorbis/vorbisfile.h>
+
+class OggStream
+{
```

Comment:
Adhere to the coding guidelines.

Reply:
I love visual studio auto-indenting.
This is generally why I prefer notepad, constantly having to correct bad auto-indents can be frustrating.
But visual studio has a compile button right there, so what's one to do.